kythe/cxx/verifier/souffle_interpreter.h

```diff
@@ -36,12 +39,15 @@ struct SouffleResult {
 /// \param inspect the inspection callback that will be used against the
 /// provided list of inspections; a false return value stops iterating through
 /// inspection results and fails the solution, while a true result continues.
+/// \param get_symbol returns a string representation of the given symbol
 /// \return a `SouffleResult` describing how the run went.
-SouffleResult RunSouffle(const SymbolTable& symbol_table,
-                         const std::vector<GoalGroup>& goal_groups,
-                         const Database& database, const AnchorMap& anchors,
-                         const std::vector<Inspection>& inspections,
-                         std::function<bool(const Inspection&)> inspect);
+SouffleResult RunSouffle(
+    const SymbolTable& symbol_table, const std::vector<GoalGroup>& goal_groups,
+    const Database& database, const AnchorMap& anchors,
+    const std::vector<Inspection>& inspections,
+    std::function<bool(const Inspection&, std::optional<std::string_view>)>
```

Comment:
What is the second optional param for? it's not documented that I can see an all uses seem to use the unhelpful parameter name `o`

Is this the output?